pkg/middleware/stats.go

```diff
@@ -24,46 +23,30 @@ func NewStats(statsClient client.Client) *Stats {
 }
 
 // Handler is the middleware function
-func (m *Stats) Handler(handler http.Handler) http.Handler {
+func (s *Stats) Handler(handler http.Handler) http.Handler {
```

Comment:
it used to be `m` because of `Middleware` - all middlewares had the same pointer name, no mater the name